cfme/services/requests.py

```diff
@@ -407,6 +407,8 @@ class network(WaitTab):  # noqa
 
     @View.nested
     class properties(WaitTab):  # noqa
+        # Cloud providers only?
```

Comment:
I'm confused by this comment, is it necessary?

Reply:
Probably not